### PR TITLE
Build application in sdk-nrf-toolchain docker image

### PR DESCRIPTION
**Pull request by jangalda-nsc**

### CODE REVIEW

**Review by karhama — APPROVED** [review]
I think these two issues also exists in build.yml

**Comment by gmarull** [comment]
Please, do not merge without further approvals. This was merged without commits following a proper structure, nor reviewed with the maintainers of this repository.

**Pull request merged**
